src/test/java/com/parasoft/findings/utils/rules/RuleDescriptionImporterTest.java

```diff
@@ -0,0 +1,130 @@
+package com.parasoft.findings.utils.rules;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RuleDescriptionImporterTest {
+
+    private static final String TEST_BUILTIN_RULES_PATH = "src/test/resources/builtinRules";
+
+    private static final String TEST_BUILTIN_RULES_2023_1_0_PATH = TEST_BUILTIN_RULES_PATH + "/2023.1.0";
+    private static final Locale DEFAULT_LOCALE = Locale.getDefault();
+
+    @AfterEach
+    void tearDown() {
+        Locale.setDefault(DEFAULT_LOCALE);
+    }
+
+    @Test
+    void testImportCpptestRuleDescription() throws Throwable {
+        Locale.setDefault(Locale.ROOT);
+        File ruleFile = new File(TEST_BUILTIN_RULES_2023_1_0_PATH, "cpptest/rules.xml");
+        List<RuleDescription> ruleDescriptions = new RuleDescriptionImporter().performImport(ruleFile);
+        assertImportCpptestRuleDescription(3828, ruleDescriptions,
+                "A macro parameter immediately following a # operator shall not immediately be followed by or preceded by a ## operator");
+    }
+
+    @Test
+    void testImportCpptestRuleDescriptionInSimplifiedChinese() throws Throwable {
+        Locale.setDefault(Locale.SIMPLIFIED_CHINESE);
+        File ruleFile = new File(TEST_BUILTIN_RULES_2023_1_0_PATH, "cpptest/zh_CN/rules.xml");
+        List<RuleDescription> ruleDescriptions = new RuleDescriptionImporter().performImport(ruleFile);
+        assertImportCpptestRuleDescription(4151, ruleDescriptions,
+                "一个紧跟在 # 操作符的宏参数不应该在其后紧跟或在其前加上 ## 操作符");
+    }
+
+    private static void assertImportCpptestRuleDescription(final int expectedRuleDescriptionCount, final List<RuleDescription> ruleDescriptions,
+                                                           final String expectedRuleDescriptionHeader) {
```

Comment:
Is it better to change name "expectedRuleDescriptionHeader" to "expectedTheFirstRuleDescriptionHeader"